plugins/repository-s3/src/main/java/org/opensearch/repositories/s3/S3BlobStore.java

```diff
@@ -211,6 +240,7 @@ public Map<Metric, Map<String, Long>> extendedStats() {
         }
         Map<Metric, Map<String, Long>> extendedStats = new HashMap<>();
         statsMetricPublisher.getExtendedStats().forEach((k, v) -> extendedStats.put(k, v.toMap()));
+        extendedStats.put(Metric.GENERIC_STATS, genericStatsMetricPublisher.stats());
```

Comment:
would PERMIT_STATS be a better name?

Reply:
But i think `GENERIC_STATS` doesn't provide the correct info around the stats bundled under this. Maybe something like `QUEUE_STATS` or `CONSUMPTION_STATS` ?

Reply:
Consumption stats are also on the same vague lines as generic and I would prefer generic to consumption to avoid confusion of what is getting consumed under the hood. And for the same previous reason queue stat doesn't fit.